draft-birkholz-scitt-receipts.md

```diff
@@ -64,7 +65,7 @@ A transparent and authentic ledger service in support of a supply chain's integr
 
 This document defines a method for issuing and verifying countersignatures on COSE_Sign1 messages included in an authenticated data structure such as a Merkle Tree.
 
-We adopt the terminology of [architecture](pointer) for Claim, Envelope, Transparency Service, Ledger, Receipt, and Verifier.
+We adopt the terminology of An Architecture for Trustworthy and Transparent Digital Supply Chains (see {{I-D.birkholz-scitt-architecture}}) for Claim, Envelope, Transparency Service, Ledger, Receipt, and Verifier.
```

Comment:
Maybe like this? This also introduces the acronym SCITT, which might come in handy.

```suggestion
We adopt the terminology of the Supply Chain integrity, Transparency, and Trust architecture document (An Architecture for Trustworthy and Transparent Digital Supply Chains, see {{I-D.birkholz-scitt-architecture}}): Claim, Envelope, Transparency Service, Ledger, Receipt, and Verifier.
```